views/apps.ejs

```diff
@@ -13,18 +13,24 @@
     <script src="https://unpkg.com/react@15/dist/react.js"></script>
     <script src="https://unpkg.com/react-dom@15/dist/react-dom.js"></script>
     <script type="text/babel">
-    const testimage = (
+
+    function Testimage(props){
```

Comment:
Lets give more appropriate component name. I think, `ImagePost` will be more appropriate for this component. Because  its a component that renders an image post. In future we might have `TextPost`, `VideoPost` etc.